python_scripts/linear_models_sol_03.py

```diff
@@ -8,14 +8,20 @@
 # %% [markdown]
 # # 📃 Solution for Exercise M4.03
 #
-# The parameter `penalty` can control the **type** of regularization to use,
-# whereas the regularization **strength** is set using the parameter `C`.
-# Setting`penalty="none"` is equivalent to an infinitely large value of `C`. In
-# this exercise, we ask you to train a logistic regression classifier using the
-# `penalty="l2"` regularization (which happens to be the default in
-# scikit-learn) to find by yourself the effect of the parameter `C`.
+# In the previous Module we tuned the hyperparameter `C` of the logistic
+# regression without mentioning that it controls the regularization strength.
+# Later, on the slides on 🎥 **Intuitions on regularized linear models** we
+# metioned that a small `C` provides a more regularized model, whereas a
+# non-regularized model is obtained with an infinitely large value of `C`.
+# Indeed, `C` behaves as the inverse of the `alpha` coefficient in the `Ridge`
+# model.
```

Comment:
For info in c1a97ab4c2e2f6fd29aafaa2c8309e0c24338622 I removed the mention to the `penalty="l2"` and `penalty="none"` as it not used anywhere else in this notebook or the MOOC, except for the first question in the [Quiz M4.05](https://inria.github.io/scikit-learn-mooc/linear_models/linear_models_quiz_m4_05.html). I suggest rather modifying the question as we also never mention what `penalty="l1"` is other than that same question.